bin/wait-for-services

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -em
+
+while true; do
```

Comment:
should there not be some sort of timeout?
how often will this check?

Reply:
indeed, it should sleep a bit.